src/main/resources/openapi/usermanagement.yaml

```diff
@@ -19,12 +19,20 @@ paths:
       parameters:
         - in: query
           name: courthouse
```

Comment:
This query param should probably be renamed to be `courthouse_id`?

Reply:
Agree, updated.